homeassistant/components/ruckus_unleashed/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Ruckus Unleashed",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "Host",
```

Comment:
Please use as many common string keys and references as possible.

https://developers.home-assistant.io/docs/internationalization/core#introducing-new-strings